build-logic/convention/src/main/kotlin/AndroidApplicationConventionPlugin.kt

```diff
@@ -15,6 +12,7 @@ private const val APPLICATION_ID = "io.homeassistant.companion.android"
  * This centralizes configuration, preventing duplication across multiple modules.
  *
  * This plugin applies several Gradle plugins that are commonly used in all application modules.
+ * More specifically it apply the [AndroidCommonConventionPlugin].
```

Comment:
Grammar + indicate that specifically only refers to build-logic and not the other plugins.

```suggestion
 * This plugin applies several Gradle plugins that are commonly used in all application modules,
 * including the [AndroidCommonConventionPlugin].
```